gui/run_manager.py

```diff
@@ -355,10 +355,9 @@ def compare_tags(self, a, b):
 
 if __name__ == "__main__":
     run_manager = RunManager()
-    new_args = {"tag": "NEW TAG NAME",
-                "args": {
-                "Kit Version": "NEW KIT VERSION"}}
-
-
-    run_manager.update_run(tag_to_find="TAG",
-                           data=new_args)
\ No newline at end of file
+    # new_args = {"tag": "NEW TAG NAME",
+    #             "args": {
+    #                 "Kit Version": "NEW KIT VERSION"}}
+    #
+    # run_manager.update_run(tag_to_find="TAG",
+    #                        data=new_args)
```

Comment:
We should remove unused code where possible.

Reply:
that should be another issue named "clean code + reformat"

Reply:
@lenhattu I disagree, but either will work since we're freezing new features.

Reply:
Are we freezing all new features next Wed?